routes/cases.js

```diff
@@ -1,6 +1,7 @@
 
 const express = require('express'),
-      router  = express.Router();
+      router  = express.Router(),
+      cases   = require('../models/models');
 
```

Comment:
Jettes un oeil la dessus:
[pass db Obj around node app](http://stackoverflow.com/questions/25670075/passing-database-connection-around-in-a-nodejs-application)